corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -640,7 +640,8 @@ def url_replace(context, field, value):
 
 @register.filter
 def view_pdb(element):
-    import ipdb; ipdb.sset_trace()
+    import ipdb
+    ipdb.sset_trace()
```

Comment:
Oh, looks like `sset_trace()` is something different - used for nose?  When running tests locally, I use some flags so I can see output while it runs - I find that much easier to debug.
```bash
alias djtest="REUSE_DB=1 ./manage.py test --nocapture --nologcapture --noinput"
```

Reply:
hey yes, `set_trace` does not work anymore, at least not for me. It just hands there and the simple fix i found was to use `sset` instead. i assumed they just changed their method call.